### PR TITLE
Issue #160 Implement `schedule delete` command

### LINKED ISSUE #160: Add `schedule delete` command
Format:
`schedule delete SCHEDULE_INDEX`
Example:
`schedule delete 2`

### PR DESCRIPTION
**Pull request by lzan98**
Closes #160

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/ScheduleCommandParser.java [hunk 1]
```diff
@@ -31,9 +32,9 @@ public ScheduleCommand parse(String args) throws ParseException {
         case "add":
             return new AddScheduleCommandParser().parse(arguments);
         /*case "edit":
-            return new EditScheduleCommandParser().parse(arguments);
+            return new EditScheduleCommandParser().parse(arguments);*/
         case "delete":
-            return new DeleteScheduleCommandParser().parse(arguments);*/
+            return new DeleteScheduleCommandParser().parse(arguments);
```
Comment (domlimm) [inline src/main/java/seedu/address/logic/parser/ScheduleCommandParser.java:37]: Weird how this is flagged out as a warning when there exists a `DeleteScheduleCommandParserTest` file already.

File: src/test/java/seedu/address/testutil/TypicalIndexes.java [hunk 2]
```diff
@@ -10,5 +10,9 @@
     public static final Index INDEX_SECOND_CANDIDATE = Index.fromOneBased(2);
     public static final Index INDEX_THIRD_CANDIDATE = Index.fromOneBased(3);
     public static final Index INDEX_FOURTH_CANDIDATE = Index.fromOneBased(4);
+    public static final Index INDEX_FIRST_INTERVIEW = Index.fromOneBased(1);
+    public static final Index INDEX_SECOND_INTERVIEW = Index.fromOneBased(2);
+    public static final Index INDEX_THIRD_INTERVIEW = Index.fromOneBased(3);
+    public static final Index INDEX_FOURTH_INTERVIEW = Index.fromOneBased(4);
```
Comment (domlimm) [inline src/test/java/seedu/address/testutil/TypicalIndexes.java:16]: Nice!

**Review by domlimm — APPROVED** [review]
All LGTM! Great job on the test cases leon and of course the functionality! 💯

File: src/main/java/seedu/address/logic/parser/ScheduleCommandParser.java (hunk 1 above)
Comment (LeeEnEn) [reply src/main/java/seedu/address/logic/parser/ScheduleCommandParser.java:37]: It is indeed weird that this is being flagged. 🤔

**Review by LeeEnEn — APPROVED** [review]
Apart from that sentence that is being flagged, all else looks good! Good job! 👍

**Pull request merged**